### ISSUE TITLE
AttributeError: 'ProgressBar' object has no attribute 'currval'

### ISSUE THREAD

**Issue by WHG555** [marcotcr/lime#420]
error code 
```
E:\Soft\anaconda\envs\retinanet\lib\site-packages\lime\lime_image.py in data_labels(self, image, fudged_image, segments, classifier_fn, num_samples, batch_size)
    263                 labels.extend(preds)
    264                 imgs = []
--> 265             pbar.currval += 1
    266             pbar.update()
    267         pbar.finish()

AttributeError: 'ProgressBar' object has no attribute 'currval'
```

Package version
```
lime:  0.1.1.37
progressbar:  2.5
progressbar2:  3.42.0
```

**Comment by MTang0728** [comment]
I have the same issue

**Comment by nishamuktewar** [comment]
When you use ProgressBar2 with newer Python versions, example 3.7, you need to update the progressbar related code in lime_image.py. The problem is that ProgressBar2 API is not backward compatible which causes the issue and hence hard to make a pull request. So have forked the repo: https://github.com/nishamuktewar/lime. You could simply copy "lime_image.py" from my forked version to your environment and it should work.

**Comment by marcotcr** [comment]
Fixed in 68c08129849cf27cc0761426e5f670b3c2e399bf by replacing progressbar with tqdm
